src/index.ts

```diff
@@ -2,3 +2,24 @@ export * from './components';
 export * from './hooks';
 export * from './models';
 export * from './utils';
+
+/**
+ * Content path to the source files in component library that utilizes Tailwind class names.
+ * This is intended to be used in user's custom tailwind config.
+ *
+ * @public
+ *
+ * @remarks
+ * This assumes that the node_modules folder containing the component library
+ * is in adjacent level with the user's tailwind.config.js file.
+ *
+ * @example
+ * In user's tailwind.config.js file:
+ *
+ * const \{ AnswersTailwindPath \} = require('\@yext/answers-react-components')
+ * module.exports = \{
+ *  content: [ AnswersTailwindPath ],
+ *  ...
+ * \};
+ */
+export const AnswersTailwindPath = 'node_modules/@yext/answers-react-components/lib/**/*.{js,ts,jsx,tsx}';
```

Comment:
can the name be changed so it doesn't have "Answers" in it? I'd rather not add more things to our libraries that reference Answers since we're re-branding